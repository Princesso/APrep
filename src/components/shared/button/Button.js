import React from 'react'
import './button.css'
import {Link} from 'react-router-dom'

const Button = () => {
  return( 
      <Link to="/login" ><button className="button" type="button">Make a Delivery Order</button></Link>  
  )
}

export default Button