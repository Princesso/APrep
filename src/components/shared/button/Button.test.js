import React from 'react';
import Button from './Button';
import renderer from 'react-test-renderer';
import {Provider} from 'react-redux';
import Store from '../../../store';
import { BrowserRouter, Route, Switch } from 'react-router-dom'

test('Button', () => {
  const component = renderer.create(
    <Provider store={Store}>
      <BrowserRouter>
          <Switch>
            <Button/>
          </Switch>
      </BrowserRouter>
    </Provider>
  )

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})