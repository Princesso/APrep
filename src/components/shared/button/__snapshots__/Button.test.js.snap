// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button 1`] = `
<a
  href="/login"
  onClick={[Function]}
>
  <button
    className="button"
    type="button"
  >
    Get Started
  </button>
</a>
`;
